pheme/templatetags/charts/treemap.py

```diff
@@ -229,7 +234,7 @@ def treemap(
     data: List[Dict],
     width=1024,
     height=768,
-    fontsize=11,
+    fontsize=16,
```

Comment:
maybe you can also provide types for these in future

```suggestion
    fontsize: int = 16,
```